mlir/include/mlir/Dialect/ArmSVE/Transforms/Passes.td

```diff
@@ -21,7 +21,7 @@ def LegalizeVectorStorage
 
     This pass currently addresses two issues.
 
-    ## Loading and storing predicate types
```

Comment:
What makes the old format "broken" and the new one "fixed"?

Reply:
At [Passes](https://mlir.llvm.org/docs/Passes/#), the TOC is broken because the pass is written in a 3 level header and the description of the pass is written in a 2 level header; the description of the pass should be in a level header below the pass, so it is modified to a 4 level header.

![スクリーンショット 2024-01-09 17 25 52](https://github.com/llvm/llvm-project/assets/5669412/c327bcf4-e58c-4bc4-91ad-7460d7ea6503)

Reply:
Thanks for the explanation!